File: data/EXP01/python/2208/2208.output_3.py
# @lc app=leetcode id=2208 lang=python3
class Solution:
    def halveArray(self, nums: List[int]) -> int:
        total_sum = sum(nums)
        nums.sort(reverse=True)
        target = total_sum / 2
        operations = 0
        
        for num in nums:
            if num <= target:
                break
            target /= 2
            operations += 1
            
        return operations

File: data/EXP01/golang_no_system_var/2208/2208.output_3.go
func halveArray(nums []int) int {
   sort.Ints(nums)
   sum := 0
   for _, num := range nums {
      sum += num
   }
   target := sum / 2
   operations := 0
   for _, num := range nums {
      for num > target {
         num /= 2
         operations++
      }
      target -= num
   }
   return operations
}
